starter-core/src/main/java/io/micronaut/starter/build/gradle/GradlePlugin.java

```diff
@@ -215,13 +214,16 @@ public GradlePlugin.Builder order(int order) {
         }
 
         @NonNull
-        public GradlePlugin.Builder requiresSettingsPluginsManagement() {
-            this.requiresSettingsPluginsManagement = true;
+        public GradlePlugin.Builder pluginsManagementRepository(GradleRepository repo) {
+            if (this.pluginsManagementRepositories == null) {
+                this.pluginsManagementRepositories = new ArrayList<>();
+            }
+            this.pluginsManagementRepositories.add(repo);
```

Comment:
Should this be a `Set`?

Different features could add the same Repo, and we'd end up with duplication in the settings file

Could probably do with a test for this as well

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1675/commits/c09bdbb3461cb73a12a629738b4a0f558cb7e919 and `MicronautGradleEnterpriseSpec`